Conceptpower+Spring/src/test/java/edu/asu/conceptpower/app/core/impl/RequestsManagerTest.java

```diff
@@ -0,0 +1,80 @@
+package edu.asu.conceptpower.app.core.impl;
+
+
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.db4o.ObjectContainer;
+
+import edu.asu.conceptpower.app.db.DatabaseManager;
+import edu.asu.conceptpower.app.db4o.IRequestsDBManager;
+import edu.asu.conceptpower.core.ReviewRequest;
+import edu.asu.conceptpower.core.ReviewStatus;
+
+@RunWith(MockitoJUnitRunner.class)
+public class RequestsManagerTest {
+    
+   
+    
+    @InjectMocks
+    ReviewRequest reviewRequest;
```

Comment:
you're not testing the review request class here, are you? Your test is called RequestsManagerTest. Why do you inject mocks?